ExportData.cs

```diff
@@ -91,48 +79,24 @@ public static void Execute(IList<IList<object>> list)
             });
 
             // Define request parameters.
+            // @todo Fetch these from a config.json file or environment variables?
             String spreadsheetId = "1-NNRsKhonKzNmTrl2H3IfwIZvGTy0HMs6AvXhsw-nUc";
+            String spreadsheetTab = "Test Data";
 
-            String range = "Test Data!A2:E";
-            //SpreadsheetsResource.ValuesResource.GetRequest request =
-            //        service.Spreadsheets.Values.Get(spreadsheetId, range);
-
-            //var list1 = new List<object> { "Hi" };
-            //var list2 = new List<object> { "How are you?" };
-
-            //IList<IList<object>> list = new List<IList<object>> { list1, list2 };
-
-            var rng = string.Format("{0}!A1:A{1}", "Test Data", list.Count);
+            // Define the sheet range
+            var rng = string.Format("{0}!A1:A{1}", spreadsheetTab, list.Count);
             var vRange = new ValueRange
             {
                 Range = rng,
                 Values = list,
                 MajorDimension = "ROWS"
             };
 
+            // Send the request to the Google Sheets API
             var rqst = service.Spreadsheets.Values.Append(vRange, spreadsheetId, rng);
             rqst.ValueInputOption = SpreadsheetsResource.ValuesResource.AppendRequest.ValueInputOptionEnum.USERENTERED;
-
             rqst.Execute();
```

Comment:
wrap in `try {} catch {}`

Reply:
Agreed. I've created an issue: https://github.com/teamtreedyn/Binoculars/issues/23 therefore marking as resolved.